spring-security-oauth2/src/main/java/org/springframework/security/oauth2/provider/endpoint/DefaultRedirectResolver.java

```diff
@@ -40,19 +40,8 @@
 
 	private Collection<String> redirectGrantTypes = Arrays.asList("implicit", "authorization_code");
 
-	private boolean matchSubdomains = true;
-
 	private boolean matchPorts = true;
 
-	/**
-	 * Flag to indicate that requested URIs will match if they are a subdomain of the registered value.
-	 * 
-	 * @param matchSubdomains the flag value to set (deafult true)
-	 */
-	public void setMatchSubdomains(boolean matchSubdomains) {
```

Comment:
`setMatchSubdomains()` cannot be removed as it can break existing applications that are using it.